TrustModel_Personas.md

```diff
@@ -0,0 +1,179 @@
+ defines Confidential Computing as the protection of data in use by performing computation in a 
+  hardware-based Trusted Execution Environment (TEE).
+
+In considering personas we recognise that a trust boundary exists between each persona and we
+explore how the least privilege principle (as described on Page 36 of
+  [Cloud Native Security Paper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf)
+) should apply to any actions which cross these boundaries.
+
+ Confidential containers can provide enhancements to ensure that the expected code/containers 
+ are the only code that can operate over the data. However any vulnerabilities within this code 
+ are not mitigated by using confidential containers, the Cloud Native Security Whitepaper 
+ details Lifecycle aspects that relate to the security of the code being placed into containers 
+ such as Static/Dynamic Analysis, Security Tests, Code Review etc which must still be followed. 
+
+![Personas Model](./images/PersonaModel.svg)
+
+Any of these personas could attempt to perform malicious actions.
+
+### Infrastructure Operator
+This persona has privileges within the Cloud Infrastructure which includes the hardware and 
+firmware used to provide compute, network and storage to the Cloud Native solution
+They are responsible for availability of infrastructure used by the cloud native environment.
+- Have access to the physical hardware.
+- Have access to the processes involved in the deployment of compute/storage/memory used by any 
+  orchestration components and by the workload.
+- Have control over TEE hardware availability/type
+- Responsibility for applying firmware updates to infrastructure including the TEE Technology.
+
+Example : Cloud Service Provider (CSP) Infrastructure as a Service (IaaS) Site Reliability Engineer
+(SRE)
+
+### Orchestration Operator
+This persona has privileges within the Orchestration/Cluster.
+They are responsible for deploying a solution into a particular cloud native environment and 
+managing the orchestration environment.
+For managed cluster this would also include the administration of the cluster control plane.
+- control availability of service
+- control webhooks and deployment of workloads
+- control availability to workload of cluster resources (data/networking/storage) and cluster 
+  services (Logging/Monitoring/Load Balancing)
+- control the deployment of boot images initially required by the TEE during initialisation. 
+ These boot images once initialised will receive the workload.
+
+Example : A kubernetes administrator responsible for deploying pods to a cluster and 
+maintaining the cluster.
+
+### Workload Provider
+This persona develops, builds and defines the solution, pods and containers intended to provide 
+the Application or Workload which in turn provides value to the data owners (customers and clients).
```

Comment:
Thanks for adding the Container Image Provider section!

A couple thoughts to work in now or in a subsequent PR...
Now that we have the Container Image Provider, we can cull some scope from the Workload Provider.

> This persona designs and creates the kubernetes objects comprising the solution. Those objects reference containers published by Container Image Providers. In some cases the Workload and Container Image Providers may be the same entity. 

i.e. the workload provider doesn't develop and build the containers since that is now in the role of the container provider.

Reply:
Agree and made some changes along the line of your suggestion - Thanks